### ISSUE TITLE
Feature request: GCP: provide toggle to disable IPv6 FW rule

### ISSUE THREAD

**Issue by JordanP** [poseidon/typhoon#475]
Global forwarding rules are not cheap on GCP. In my case, they account for just a bit less than half the cost for a 3 n1-standard-1 premptible nodes cluster.

An extra Ipv6 address and forwarding rule was introduced in https://github.com/poseidon/typhoon/commit/5be5b261e2a6a4c7628b80a2ae3deb626ba7a945 Would you accept a PR that introduce a variable/toggle to optionally not create those resources ?

**Comment by dghubble** [comment]
https://twitter.com/dghubble/status/1074186919087468544

I'm keenly aware GCP charges this way and I agree its annoying. I pay for my own clusters, so I get it. But I designed it this way because I believe IPv6 adoption is important enough. I have ensured only 5 global forwarding rules are created - since GCP charges for the first batch of 5, its the same cost for folks with one cluster in their project.

As for additional variables to disable IPv6, I'd prefer not to for several reasons. First, my aim is to go in the opposite direction so that Ingress is exposed via an IPv4 and IPv6 address on every platform, consistently and simply out of the box. I don't want to create a situation where someone chooses between IPv6 adoption and cutting costs. I disagree with GCP's decision to charge separately for IPv6 rules and dislike the idea of adding complexity here to accomodate their decisions (e.g. Google Cloud has the resources to be encouraging IPv6 adoption here, but instead the asks bubble up to this project to add/document special casing and possibly turn off IPv6). Finally, if any forwarding rules were to be made optional, it should be the IPv4 ones over the very long term.
